RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1741,6 +1772,25 @@ boolean L11_hiddenCity()
 		if(!elevatorAction)
 		{
 			auto_log_info("Hidden Apartment Progress: " + get_property("hiddenApartmentProgress"), "blue");
+
+			int turnsUntilElevatorAction = zone_delay($location[The Hidden Apartment Building])._int;
+
+			if(auto_have_familiar($familiar[Nosy Nose]) && auto_is_valid($skill[Get a Good Whiff of This Guy]))
+			{
+				if(have_effect($effect[Thrice-Cursed]) < (turnsUntilElevatorAction + 1)  && 
+				appearance_rates($location[The Hidden Apartment Building])[$monster[pygmy shaman]] < 100)
+				{
+					handleFamiliar($familiar[Nosy Nose]);	//whiff increases chance of shamen. the deleveling can also help survive being cursed
+				}
+				else if(appearance_rates($location[The Hidden Office Building])[$monster[pygmy witch accountant]] >= 20 &&	item_amount($item[McClusky file (complete)]) == 0)
```

Comment:
the default rate should be 20% or better and accountants should not get banished I guess it was in case shaman was olfacted then there's too low a chance of accountants and the nose could even be hindering them?